Modix.Data/Models/Core/MessageEntity.cs

```diff
@@ -22,6 +22,8 @@ public class MessageEntity
         [Required]
         public DateTimeOffset Timestamp { get; set; }
 
+        public ulong? StarboardEntryId { get; set; }
```

Comment:
Does it make sense to make this a foreign key to Messages and to add a navigation property for the same?